src/asdf/install.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+echo "Activating asdf ..."
+
+if [ "$(id -u)" -ne 0 ]; then
```

Comment:
Better explicit the value than having magic number

```suggestion
ROOT_USER_ID=0
if [ "$(id -u)" -ne $ROOT_USER_ID ]; then
```

Reply:
taken into account